edx_toggles/toggles/internal/waffle/flag.py

```diff
@@ -92,13 +86,12 @@ def _get_flag_active_no_request(self):
         set_custom_attribute("warn_flag_no_request_return_value", value)
         return value
 
-    def set_monitor_value(self, value):
+    def set_monitor_value(self, _value):
         """
-        Send waffle flag value to monitoring. We keep this method such that it can be called by child classes (such as
-        edx-platform's waffle_utils.CourseWaffleFlag). A child class is expected to call this method if it is
-        overriding `is_enabled`, and has cases where it is not calling `super().is_enabled()`.
+        This used to send waffle flag values to monitoring, but is now a no-op. This method is preserved for backward
+        compatibility.
         """
-        _set_waffle_flag_attribute(self.name, value)
+        set_custom_attribute("deprecated_WaffleFlag_set_monitor_value", self.name)
```

Comment:
This may be the only one, but just in case, I'd like to limit dashboard updates.  Could we make this something like?:
```
        set_custom_attribute(
            "deprecated_waffle_method",
            "WaffleFlag[{}].set_monitor_value".format(self.name),
        )
```